client/web/src/repo/settings/components/ActionContainer.tsx

```diff
@@ -81,7 +91,7 @@ export class ActionContainer extends React.PureComponent<Props, State> {
                         <Tooltip content={this.props.buttonSubtitle}>
                             <Button
                                 className={classNames(styles.btn, this.props.buttonClassName)}
-                                variant={this.props.buttonClassName ? undefined : 'primary'}
+                                variant={this.props.buttonClassName ? this.props.buttonClassName : 'primary'}
```

Comment:
Based on the comment here.

```suggestion
                                variant={this.props.buttonClassName ? undefined : 'primary'}
```